src/database/DataTypes/VaccineVialMonitorStatus.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Sustainable Solutions (NZ) Ltd. 2020
+ * mSupply Mobile
+ */
+
+import Realm from 'realm';
+
+export class VaccineVialMonitorStatus extends Realm.Object {}
+
+VaccineVialMonitorStatus.schema = {
+  name: 'VaccineVialMonitorStatus',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    description: { type: 'string', optional: true },
+    code: { type: 'string', optional: true },
+    level: { type: 'double', default: 0 },
+    shouldDispense: { type: 'bool', default: false },
+    isActive: { type: 'string', default: false },
+  },
```

Comment:
so: 
- `level` is to be able to order the status' when displaying. But also I am realizing we also want to assosciate an icon with it.. so.. ideas?
-`shouldDispense` is I think not needed because we're not auto dispensing?
- `isActive` is just for your standard soft deletion mechanism for system data.. seems better to have it mandatory for any system data rather than coming back in 1-2 years to do it..
- `description`/`code`.. self explanatory.. but .. needed?

Are these going to be user defined or system defined ? We could allow for a choice maybe, "use 4 level system" or something if we ever need ?

Reply:
probs not for me to comment on.. but fwiw:
- code isn't very helpful, as level is more of an identifier for the status
- some sort of mapping for icons? have an icon with a property of `level` to match them up? or an `icon` prop
- can't see why `shouldDispense` is needed
- system defined makes sense to me 
- a way of translating the descriptions would be helpful

😃

Reply:
> a way of translating the descriptions would be helpful

Hmm.. maybe using `code`?

Reply:
pfft..! you and your crazy ideas 🙄